documentation/Set-PnPTenant.md

```diff
@@ -1353,7 +1353,7 @@ Accept wildcard characters: False
 ```
 
 ### -DisabledModernListTemplateIds
-Guids of out of the box modern list templates to hide when creating a new list
+Guid's of out of the box modern list templates to hide when creating a new list
```

Comment:
The previous version is the correct one.

Reply:
updated